lib/services/inbound_mail_parser.rb

```diff
@@ -0,0 +1,48 @@
+class InboundMailParser
```

Comment:
this service class is covered by feature specs for `Handler.rb` - would be great to have unit tests too and then just stub this when testing `Handler.rb` but this PR felt like it was already getting pretty big